_maps/RandomRuins/SpaceRuins/crashedship.dmm

```diff
@@ -240,11 +238,11 @@
 /area/awaymission/bmpship/aft)
 "lx" = (
 /obj/structure/cable,
-/obj/machinery/power/apc/unlocked{
-	dir = 1;
+/obj/machinery/power/apc/auto_name/directional/north{
 	environ = 0;
-	pixel_y = 25
+	
 	},
```

Comment:
honestly bro i would discard all your map edits and then run the updatepaths because that's the only reason you're getting weird shit like this

Reply:
ah never mind you pushed a commit to fix this as i was reviewing it . keep it in mind for next time though.

Reply:
I'll remake it in a sec

Reply:
odds are you've gotten everything but lets just be sure be doing a rerun yea

Reply:
Done